google/cloud/security/notifier/notifier.py

```diff
@@ -152,7 +152,28 @@ def process(message):
             payload.get('email_description'))
         return
 
-# pylint: disable=too-many-branches
+def run_cscc_notification(notifier_configs, violations_as_dict):
+    """Get the first class in the given sub module
+
+    Args:
+        notifier_configs (dict): Notification configuration.
+        violations_as_dict (tuple): Tuple of violations in dict format.
+    """
+    if not notifier_configs.get('violation'):
+        LOGGER.debug('Violation section is not found in forseti config file.')
+        return
+    if not notifier_configs.get('violation').get('cscc').get('enabled'):
+        LOGGER.debug('CSCC notification is not enabled.')
+        return
+    if '{CSCC_BUCKET}' in (
+            notifier_configs.get('violation').get('cscc').get('gcs_path')):
+        LOGGER.debug('CSCC bucket is not configured.')
+        return
+    CsccPipeline().run(
+        violations_as_dict,
+        notifier_configs.get('violation').get('cscc').get('gcs_path'))
```

Comment:
I think we have the same problem here, if the user didn't update their config file, this will throw NoneType doesn't have attribute exception.

Reply:
Never-mind, you checked it at the beginning.